src/animation.cpp

```diff
@@ -20,7 +20,7 @@ point relative_view_pos(player const &u, point const &p) noexcept
 
 void draw_animation_delay(long const scale = 1)
 {
-    auto const delay = static_cast<long>(OPTIONS["ANIMATION_DELAY"]) * scale * 1000000l;
```

Comment:
Very well may be, I think I read in searching about this oddity that gcc has no issues with it, just lvm/clang.